index.html

```diff
@@ -2318,6 +2318,22 @@ <h1>Speakers</h1>
                             </div>
                           </div>
 
+                           <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Tiejun Chen" src="img/tiejun.jpg">
```

Comment:
Follow naming convention for image. For this use -  'TiejunChen.jpg' and make a reference from this PR to the respective issue, provide deployment link. Do also delete the reference in the respective issue because which is pointing to the already closed PR.

Reply:
Okay!